### ISSUE TITLE
Possibly displaying non-breaking spaces.

### ISSUE THREAD

**Issue by BitOfAByte** [michael-lazar/rtv#170]
When browsing [Bruce Campbell's AMA](https://www.reddit.com/r/IAmA/comments/3saj8z/im_bruce_campbell_im_back_ask_me_anything/) I noticed that quite a few of his posts displayed `&nbsp;`, the non-breaking space. I'm not sure if this is just the system I was using it through, or an overall problem with rtv.

Here is an example of what I'm talking about:

>ImBruceCampbell • 66 pts 25min
Playing Reagan in Fargo was great. My buddy, John Cameron, who produces the show - we’re high school buddies. We go all the way back and my kids grew up in the Reagan era during my formative years, so John and I would imitate him mercilessly because he was a very “imitatible” president. His mannerisms, the way he spoke and everything – and you know, look, every actor wants to do a couple of things in life. Like, I’ve played Elvis. Now I’ve done Ronald Reagan. I’ve played a cowboy.

>`&nbsp;`

>Those are some of the bucket list things that actors do because they want to play certain iconic roles. The one thing I had to do was take it seriously: a) because it’s Fargo; they win Emmy’s and b) because there would be Ronald Reagan fans that would chase me down and beat me with sticks if I made him look like a jerk, or a moron, or whatever. The trick is to humanize the man.

**Comment by michael-lazar** [comment]
Hi,

This has been known to happen from time to time with Reddit's API. See https://github.com/michael-lazar/rtv/issues/96. Basically if you look at the API response it looks like this

https://www.reddit.com/r/IAmA/comments/3saj8z/im_bruce_campbell_im_back_ask_me_anything/.json
```
My buddy, John Cameron, who produces the show - we\u2019re high school buddies. We go all the way back and my kids grew up in the Reagan era during my formative years, so John and I would imitate him mercilessly because he was a very \u201cimitatible\u201d president. His mannerisms, the way he spoke and everything \u2013 and you know, look, every actor wants to do a couple of things in life. Like, I\u2019ve played Elvis. Now I\u2019ve done Ronald Reagan. I\u2019ve played a cowboy. \n\n&amp;nbsp;\n\nThose are some of the bucket
```
Reddit is actually sending us a literal ``&amp;nbsp;``, which we correctly decode as ``&nbsp;`` To get around this we would have to parse the html twice. I was originally against this, if just out of the principle, but it seems like reddit is incapable of fixing the problem so it might be worth doing.

**Comment by michael-lazar** [comment]
Resolved here 9de6056b0413c793279ef35d6ac7cd4a5a8ab38a. We're now parsing the html twice.
